--- Python/setup.py	2024-03-26 18:05:47.084806282 +0000
+++ Python/setup_patched.py	2024-03-26 18:08:09.870698787 +0000
@@ -504,7 +504,7 @@
 
 setup(
     name="pytigre",
-    version="2.4.0",
+    version="2.6.0",
     author="Ander Biguri, Reuben Lindroos, Sam Loescher",
     packages=find_packages(),
     include_package_data=True,
@@ -512,7 +512,6 @@
     ext_modules=[Ax_ext, Atb_ext, tvdenoising_ext, minTV_ext, AwminTV_ext, gpuUtils_ext, RandomNumberGenerator_ext],
     py_modules=["tigre.py"],
     cmdclass={"build_ext": BuildExtension},
-    install_requires=["Cython", "matplotlib", "numpy", "scipy", "tqdm"],
     license_files=("LICENSE",),
     license="BSD 3-Clause",
     # since the package has c code, the egg cannot be zipped
